Tools/Scripts/webkitpy/browserperfdash/browserperfdash_runner.py

```diff
@@ -311,6 +313,11 @@ def run(self):
                                                     self._args.browser,
                                                     None,
                                                     browser_args=self._args.browser_args)
+                    if self._args.query_browser_version:
+                        self._result_data['browser_version'] = runner._browser_driver.browser_version()
+                        if not self._result_data['browser_version']:
+                            raise NotImplementedError('The driver for browser {browser} does not implement a way to automatically obtain the version. Please specify the version manually.'.format(browser=self._args.browser))
```

Comment:
I would use f strings in modern Python.
`raise NotImplementedError(f'The driver for browser {self._args_browser}....)`

Reply:
yes, for consistency with the surrounding code I used `format()` in this case